### DISCUSSION TITLE
Office Hours March 17, 2023

### DISCUSSION THREAD

**Discussion by dcherian** [pydata/xarray#7564] [Office Hours]
High-level discussion: https://github.com/pydata/xarray/discussions/7216

## Announcement

**Registration:** https://www.eventbrite.com/e/xarray-virtual-office-hours-2023-tickets-561001549737

**When:** Friday March 17, 2023 7AM-9AM Pacific Time

**Where:** Online (Zoom link to be provided to those registered). 

**Cost:** Free!

**Who:** Will cap this first session at 20 people so that we can ensure small group interactions and utilize breakout rooms for screen sharing and pair programming 

**What:** Get to know developers and users of Xarray and have an opportunity to troubleshoot issues and learn new skills! We are hosting a virtual office hour session via Zoom. Anyone is welcome to join, for all or part of the session. We encourage you to come with a specific question or piece of code to work on, for example, is any Xarray documentation (https://docs.xarray.dev/) or tutorial ([https://tutorial.xarray.dev](https://tutorial.xarray.dev/intro.html)) material challenging to understand? Do you have questions about adopting examples to your own data analysis? 
Please also consider posting specific questions on Xarray’s Discussions page! https://github.com/pydata/xarray/discussions. 

Let’s make progress and grow the fantastic Xarray community together! 

Please review our Code of Conduct before the event! (https://github.com/pydata/xarray/blob/main/CODE_OF_CONDUCT.md)

**Comment by dcherian** [comment]
Helpers please like this post to lets us know you're available to help out.

**Reply by banesullivan** [reply]
I’d love to attend and help out if questions arise with pyvista-xarray and/or 3D viz

plus I’d love to meet many of you all!

**Reply by dcherian** [reply]
Thanks @banesullivan !

Interesting. We haven't advertised any specific topics, but I can totally see that being useful. 

cc @scottyhq : shall we update the eventbrite description? I can tweet it out now, and see if someone shows up.
